forge-gui/res/cardsfolder/upcoming/calix_guided_by_fate.txt

```diff
@@ -0,0 +1,11 @@
+Name:Calix, Guided By Fate
+ManaCost:1 G W
+Types:Legendary Enchantment Creature Human Druid 
+PT:2/2
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self,Enchantment.YouCtrl+Other | TriggerZones$ Battlefield | Execute$ TrigPutCounter | TriggerDescription$ Constellation — Whenever CARDNAME or another enchantment enters the battlefield under your control, put a +1/+1 counter on target creature.
+SVar:TrigPutCounter:DB$ PutCounter | ValidTgts$ Creature | CounterType$ P1P1 | CounterNum$ 1 
+T:Mode$ DamageDone | ValidSource$ Card.Self,Creature.enchanted+YouCtrl | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigToken | ActivationLimit$ 1 | TriggerDescription$ Whenever CARDNAME or an enchantment creature you control deals combat damage to a player, you may create a token that's a copy of a nonlegendary enchantment you control. Do this only once each turn 
```

Comment:
Wrong source

Reply:
Messed up the spelldescription 
![image](https://user-images.githubusercontent.com/67333662/233727490-e5964415-b013-40aa-8512-a1cc7b9e4bb1.png)

Reply:
I think he means that ValidSource$ Card.Self,Creature.enchanted+YouCtrl needs to be changed toValidSource$ Card.Self,Creature.enchantment+YouCtrl. Plus you forgot the periods on both descriptions.